pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/ReadableSequentialData.java

```diff
@@ -28,7 +29,7 @@ public interface ReadableSequentialData extends SequentialData {
      *
      * @return The signed byte at the current {@link #position()}
      * @throws BufferUnderflowException If there are no bytes remaining in this sequence
-     * @throws DataAccessException If an I/O error occurs
+     * @throws UncheckedIOException If an I/O error occurs
```

Comment:
It doesn't change a lot, but if you're wanting to document this, you can add the exeception in an actual `throws` clause on the method as well.  Doing that can help callers be aware of the exception without reading Javadoc (which many people don't look at very often).

Reply:
I wish this wasn't true. I wish people actually read the documentation.

I may not fully agree with this suggestion because at first glance one could think that these checked exceptions when they see a method signature with the `throws` clause. Also, we have to mention the exceptions at least twice now, which isn't nice. But I implemented this in the latest commit. Please have a look.